pkg/havener/exec.go

```diff
@@ -87,13 +88,33 @@ func TraverseStructureAndProcessOperators(input interface{}) (interface{}, error
 			return nil, err
 		}
 
+		if isPotentialBoolean(input) {
+			input, _ = strconv.ParseBool(input.(string))
+		}
+
 	case nil:
 		input, err = map[interface{}]interface{}{}, nil
 	}
 
 	return input, err
 }
 
+// isPotentialBoolean checks if the argument looks like
+// a false/true bool. This is not supported by default
+// in shell env variables. But, is a way to allow users
+// to define booleans in the values.yml, where booleans
+// are supported by helm
+func isPotentialBoolean(input interface{}) bool {
+	shellRegexp := regexp.MustCompile(`(false|true)`)
```

Comment:
I would change the Regex expression to `^(false|true)$` (https://regex101.com/r/2hrxa6/1) because else it also matches entire sentences which contain the words _true_ or _false_ (see link). However this would cause an error in line 92 even if this scenario is pretty rare.

Reply:
good finding, thanks. Fixed with a new commit